tests/drivers/dma/loop_transfer/boards/lpcxpresso55s36.overlay

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright C 2023 NXP
```

Comment:
Sorry, missed this- please fix the copyright string here

Reply:
Apologies, this is fixed now